include/sir/platform.h

```diff
@@ -446,6 +451,24 @@ typedef BOOL(CALLBACK* sir_once_fn)(PINIT_ONCE, PVOID, PVOID*);
 #  error "unable to resolve thread local attribute; please contact the author."
 # endif
 
+# if defined(__WIN__) && defined(__ORANGEC__)
+#  if defined(__ORANGEC_MAJOR__) && defined(__ORANGEC_MINOR__) && defined(__ORANGEC_PATCHLEVEL__)
+#   if __ORANGEC_MAJOR__ <= 6
+#    if __ORANGEC_MINOR__ <= 70
+#     if __ORANGEC_PATCHLEVEL__ <= 92
+#      if !defined(ORANGEC_VERSION_WARNING)
+#       warning OrangeC versions before 6.0.70.93 are unsupported.
+#       define ORANGEC_VERSION_WARNING
```

Comment:
Shouldn't we just #error here? what's the point of warning and continuing?

Reply:
Earlier versions can be made to work, but they're missing some functionality like strnlen - up to you. :)